test/tests_for_integration/authjars

```diff
@@ -0,0 +1,74 @@
+kbase/auth2/kbase-auth2templates-0.4.3.zip
```

Comment:
was this file generated by hand or autogenerated in some manner?

Reply:
Some of both. I use https://github.com/kbase/auth2/blob/master/build.xml#L143-L146 to get the class path and then format / rearrange / comment. I could probably add an ant task to do a bit more of it for me but it changes so rarely the ROI probably isn't there.